apps/sap-commerce-cloud/src/api/fetchProductPreviews.ts

```diff
@@ -23,14 +23,9 @@ export async function fetchProductPreviews(
         },
       };
   for (const sku of skus) {
-    const splitSku = sku.split(':');
-    const baseSite = splitSku[0];
-    const skuId = splitSku[1];
-    skuIds.push(skuId);
+    skuIds.push(sku.split('/products/').pop()!);
```

Comment:
Curious what the `!` is for at the end of the function. Is this an assertion? (not blocking)

Reply:
@pkeavenycontentful https://github.com/contentful/apps/pull/3062#discussion_r1156372871 😂 

For what it's worth I think this is pretty tough to read and the fact two of us weren't sure what it did makes me think it's likely to be misunderstood again. Maybe we could rework this slightly to do a quick check on that string before attempting a pop. More lines, but maybe more readable?

Reply:
Yeah this is a non-null assertion that was added in TypeScript 2.0. In this case we can safely assume this will always be defined. 

I can add a check to throw an error instead to keep the same sentiment 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
Changed it to use `as string` to keep with the sentiment of "this will not ever be undefined"